### PR TITLE
Update the branch into client-go for the package jcrossley3/manifestival

### PR DESCRIPTION
**Pull request by houshengbo**
The purpose of this PR is to get rid of the usage of client.Client in controller-runtime.

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -19,14 +19,15 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"k8s.io/client-go/tools/clientcmd"
```
Comment (k4leung4) [inline cmd/manager/main.go]: nit: group with other k8s import

Comment (houshengbo) [reply cmd/manager/main.go]: done

**Comment by greghaynes** [comment]
I really dont think this should pull in 85k of vendor updates? Is it possible you ran update-deps while this was checked out in the old (github.com/knative/serving-operator) path?

**Comment by greghaynes** [comment]
@jcrossley3 Do you want to keep this as a separate branch in manifestival repo or are you interested in making this master or?

**Comment by houshengbo** [comment]
@greghaynes When we change the branch from master to client-go for github.com/jcrossley3/manifestival, all the changes happened accordingly, if we run ./hack/update-deps.sh.

**Pull request closed without merging**